test/lib/validation.spec.ts

```diff
@@ -53,7 +54,11 @@ const scenarios: Scenario[] = [
   { type: 'dob', prop: '2030-05-25', isValid: false },
   { type: 'dob', prop: '2016-05-25', isValid: true, altered: '2016-05-25' },
   { type: 'dob', prop: ' 20 16- 05- 25 ', isValid: true, altered: '2016-05-25' },
-
+  { type: 'dob', prop: '20', isValid: true, altered: DateTime.now().minus({ years: 20 }).toISODate() },
```

Comment:
should the tests mock now and hardcode the expected output?

Reply:
I don't see the value if mocking `now` here, this mirrors the code that will run in production more. Unless we're thinking of changing the date time lib?